src/languageServerInstaller.ts

```diff
@@ -168,14 +204,41 @@ export class LanguageServerInstaller {
 		});
 	}
 
+	verify(release: { builds?: any[]; version: any; shasums?: any; shasums_signature?: any; }, pkg: string, buildName: string) {
+		return new Promise<void>((resolve, reject) => {
+			const hash = crypto.createHash('sha256');
+			const pkgStream = fs.createReadStream(pkg);
+
+			pkgStream.on('data', (data) => {
+				hash.update(data);
+			});
+
+			let shasumResponse = "";
+			https.get(`${releasesUrl}/${release.version}/${release.shasums}`, (response) => {
+				response.on('data', (data) => {
+					shasumResponse += data;
+				});
+				response.on('end', () => {
+					const shasumLine = shasumResponse.split(`\n`).find(line => line.includes(buildName));
+					if (!shasumLine) {
+						return reject(`Install error: no matching SHA sum for ${buildName}`);
+					}
+					const shasum = shasumLine.split(" ")[0];
```

Comment:
I'm assuming this is fine, given that you only need the first part after splitting, but FYI the checksum files have 2 spaces between the sum and filename, at least when released by GoReleaser: https://github.com/goreleaser/goreleaser/blob/fd7dbf65922663f912e6561e94aecc7d2a83c3c0/internal/pipe/checksums/checksums.go#L87

Reply:
Fixed it to avoid confusion later.